src/components/expression/expression-loader.js

```diff
@@ -133,40 +148,103 @@ genotet.ExpressionLoader.prototype.loadExpressionMatrix_ = function(fileName,
       });
     this.data.lowerConditionNames = lowerConditionNames;
 
-    this.loadTfaProfile_(this.data.tfa.fileName, this.data.matrix.geneNames,
-      this.data.matrix.conditionNames);
+    if (this.data.profile.geneNames.length != 0) {
```

Comment:
Sry it's not negation. But you could have got what I mean
```js
if (this.data.profile.geneNames.length) {
}
```
Do not compare length with 0.